### DISCUSSION TITLE
Change DNS server

### DISCUSSION THREAD

**Discussion by stenosis** [OnionUI/Onion#1536] [Q&A] [answered]
Hi there,

I'm wondering if it's possible to change the default DNS server permanently?

Changes to the resolv.conf inside the /appconfigs or /etc folder get's reversed on restart and I can't find some sort of network manager who causes this.

**Comment by XK9274** [comment, marked as answer]
https://github.com/OnionUI/Onion/issues/1500

Thanks for the message.

We'll be adding a new WiFi menu at some point as we now have a global keyboard.

Udhcpc is likely changing your DNS based on what it receives from DHCP. You could try changing the DNS that your router provides to the device through the router itself if it has this option.

**Comment by stenosis** [comment]
Ah, I see.. it was udhcpc that overwrote the value - that was the clue I was looking for, thanks and keep up the good work on the Onion project .-)
